src/Illuminate/Support/Stringable.php

```diff
@@ -722,6 +723,16 @@ public function dd()
         exit(1);
     }
 
+    /**
+     * Convert the object to a string when JSON encoded.
+     *
+     * @return string
+     */
+    public function jsonSerialize()
+    {
+        return $this->value;
```

Comment:
Can we use the `__toString()` magic method here like in Mior's PR? https://github.com/laravel/framework/pull/35680/files#diff-3e127621d600522d2ea38948b7f7dc80f875966049aba26391c901efe0eb59d4R733

Reply:
Yeah, totally. I don't see the value in using `__toString()` though. In my mind, it's just causing more work in PHP. Enlighten me? 🙏

Reply:
Well the method is already there and it just prevents two methods from doing the same thing. Maybe @crynobone can answer this better on why he opted for this approach?

Reply:
I did it that way just to avoid any regression in the future if somewhere in future `$this->value` can become something other than scalar string because Laravel doesn't use strict type. Right now the safeguard is only on the constructor where we cast `(string) $value`